oct/config.sh

```diff
@@ -0,0 +1,10 @@
+AWK=$(which awk)
+BZIP2=$(which bzip2)
+FILE=$(which file)
+GZIP=$(which gzip)
+# TODO: Get this automatically.
+LIBNXZ="LD_PRELOAD=$(pwd)/libnxz.so.0.61.3 LD_LIBRARY_PATH=$(pwd) ./minigzipsh"
```

Comment:
I modified the libnxz library path.
I'm also documenting the steps to setup oct tests in the README.md as suggested.
However, I disagree with the Makefile suggestion.

IMHO, oct/config.sh should not even exist in the repository.  It should be created automatically by a configure script.  In that ideal scenario, the Makefile doesn't need to care about the contents of the variables defined in oct/config.sh.
I completely agree with you it's frustrating to fail all the tests because the directory was not properly configured, but trying to catch this in the Makefile would just workaround the real issue.

@rzinsly Do you agree to fix this in a configure script later?

Reply:
Ok, good enough for me.